start-all.sh

```diff
@@ -578,7 +578,19 @@ else
 fi
 
 if [ "$TARGET_DIRECTORY" != "" ]; then
-    SCYLLA_TARGET_FILE="-v "$($readlink_command $TARGET_DIRECTORY)":/etc/scylla.d/prometheus/targets/"
+    SCYLLA_TARGET_FILE="-v "$($readlink_command $TARGET_DIRECTORY)":/etc/scylla.d/prometheus/targets/:z"
+    if [ ! -f $TARGET_DIRECTORY/scylla_servers.yml ]; then
+        echo "Warning, using $TARGET_DIRECTORY for Prometheus traget directory, scylla_servers.yml is missing, make sure to create it, or ScyllaDB targets will be missing"
+    fi
+    if [ ! -f $TARGET_DIRECTORY/node_exporter_servers.yml ]; then
+        echo "Warning, using $TARGET_DIRECTORY for Prometheus traget directory, node_exporter_servers.yml is missing, make sure to create it, or node-exporter targets will be missing"
+    fi
+    if [ ! -f $TARGET_DIRECTORY/scylla_manager_agents.yml ]; then
```

Comment:
If one does *not* use Manager, this warning is rudendent.
Its look like Prometheus do work without Manager agent or server

Reply:
I'll make the manager optional (or add an option to by pass it) in the future, then it will be ok not to add the files, I'm trying to avoid the situation that something is wrong with the setup but there is no indication to the user